cockatrice/src/client/network/replay_timeline_widget.h

```diff
@@ -18,6 +18,9 @@ class ReplayTimelineWidget : public QWidget
 
 private:
     QTimer *replayTimer;
+    static constexpr int BASE_REWIND_BUFFERING_TIMEOUT_MS = 180;
+    static constexpr int MAX_REWIND_BUFFERING_TIMEOUT_MS = 280;
```

Comment:
Any context on where these magic numbers came from?

Reply:
I just fiddled with the numbers until it felt good enough

Reply:
they could become configurable, I'm good with merging this as is though